src/controllers/abstract-controller.ts

```diff
@@ -1,9 +1,21 @@
 import config from 'config';
 import { Router, NextFunction, Request, Response } from 'express';
 import { Repository } from 'typeorm';
+import { validate } from 'class-validator';
 
 const UUID_LENGTH = config.get('UUID_LENGTH');
 
+class ValidationError {
```

Comment:
Or simply reuse CustomError from utils, or move it in utils

Reply:
Also, as you had touched Abstract Controller please change native Errors to Custom in checkId

Reply:
Thank you, I've missed existence of CustomError. Done.